plugins/git/README.md

```diff
@@ -10,184 +10,186 @@ plugins=(... git)
 
 ## Aliases
 
-| Alias                | Command                                                                                                                          |
-|:---------------------|:---------------------------------------------------------------------------------------------------------------------------------|
-| g                    | git                                                                                                                              |
-| ga                   | git add                                                                                                                          |
-| gaa                  | git add --all                                                                                                                    |
-| gapa                 | git add --patch                                                                                                                  |
-| gau                  | git add --update                                                                                                                 |
-| gav                  | git add --verbose                                                                                                                |
-| gap                  | git apply                                                                                                                        |
-| gapt                 | git apply --3way                                                                                                                 |
-| gb                   | git branch                                                                                                                       |
-| gba                  | git branch -a                                                                                                                    |
-| gbd                  | git branch -d                                                                                                                    |
+| Alias                | Command                                                                                                                                     |
+|:---------------------|:--------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
It's not necessary to change this line at all; markdown will render the table properly as-is. (Because every line in this table has changed here, it's hard to see what the intended changes actually are.)
```suggestion
|:-|:-|
```

Reply:
I see. Alright, I've updated it. It should be fixed now